packages/terra-tag/src/_SharedUtil.js

```diff
@@ -0,0 +1,22 @@
+class SharedUtil {
+  /**
+   * Util to determine if the user agent indicates that the browser is Safari
+   * @return {boolean}
+   */
+  static isSafari() {
+    if (navigator.userAgent.indexOf('Safari') !== -1 && navigator.userAgent.indexOf('Chrome') === -1) {
+      return true;
+    }
+
+    return false;
+  }
+
+  /**
+   * Util to determine if the user agent indicates that it is macOS
+   * @return {boolean}
+   */
+
+  static isMac = () => navigator.userAgent.indexOf('Mac') !== -1 && navigator.userAgent.indexOf('Win') === -1;
```

Comment:
I cannot find where isMac is used.  I am also curious why we need this file.  What logic is really specific only to Safari?

Reply:
Removed IsMac as no references found. Thanks